src/state/assets/userAssets.ts

```diff
@@ -451,49 +498,48 @@ export const createUserAssetsStore = (address: Address | string) =>
               hiddenAssets.add(uniqueId);
             }
           });
-          return { hiddenAssets };
+
+          let hiddenAssetsBalance = '0';
```

Comment:
below we're using a number instead of a string.  Seems like here it should be a number too.

Reply:
It uses the `BigNumber` utils so it's simpler to initialize it as a string, since `add` here returns a string:

https://github.com/rainbow-me/rainbow/blob/59cb998aead12c9462dcef81abca8ed73e5ca619/src/state/assets/userAssets.ts#L467